vm-setup/roles/v1a2_integration_test/tasks/verify.yml

```diff
@@ -0,0 +1,43 @@
+---
+  - name: Waiting for controlplane to come up
+    wait_for:
+      host: #WIP
+      port: 22
+      state: started
+      timeout: 1200
+      delay: 10
+    delegate_to: 127.0.0.1
+    when: node_ip.rc == 0
+
+  - name: Check if k8s pods in running state
+    shell: "kubectl get pods --all-namespaces | grep Running | wc -l"
+    environment:
+      KUBECONFIG: '{{ KUBECONFIG_PATH }}'
+    retries: 150
+    delay: 3
+    register: k8s_running_pods
+    failed_when: >
+      (k8s_running_pods.stderr != "") or
+      (k8s_running_pods.rc !=0)
+    delegate_to: #WIP
+    until: k8s_running_pods.rc !=0
+
+  - name: Check if k8s nodes in ready state
+    shell: "kubectl get nodes | grep Ready | wc -l"
```

Comment:
Careful here! You are doing grep Ready, but it actually matches "NotReady" because we don't have a CNI installed